From 0783df8ee30fff62827490913e469200721d5187 Mon Sep 17 00:00:00 2001
Message-Id: <0783df8ee30fff62827490913e469200721d5187.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Tue, 28 Oct 2014 17:13:59 +0000
Subject: [PATCH 03/10] REVERTME [VPG]: drm/i915: Fix uninitialized return
 type in exec list code

Fix fumble in the port of the following patch:

commit 99d0ff9bd40efd88a03f0b401015387660844ff9
Author: ian-lister <ian.lister@intel.com>
Date:   Mon Dec 23 12:23:50 2013 +0000

    REVERTME [VPG]: drm/i915: Watchdog timeout on individual batch buffers

For: GMINL-1778
Change-Id: I14176162d07ee25367ba2ffc05103691347edff4
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_lrc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_lrc.c b/drivers/gpu/drm/i915/intel_lrc.c
index 4fddb7e..da0aa5f 100644
--- a/drivers/gpu/drm/i915/intel_lrc.c
+++ b/drivers/gpu/drm/i915/intel_lrc.c
@@ -851,7 +851,7 @@ int intel_execlists_submission(struct drm_device *dev, struct drm_file *file,
 
 	if (args->flags & I915_EXEC_GEN7_SOL_RESET) {
 		DRM_DEBUG("sol reset is gen7 only\n");
-                return ret;
+		return -EINVAL;
 	}
 
 	ret = logical_ring_alloc_seqno(ring, ringbuf->FIXME_lrc_ctx);
-- 
1.7.9.5

